railties/lib/rails/generators/rails/sessions/templates/controllers/concerns/authentication.rb

```diff
@@ -0,0 +1,64 @@
+module Authentication
```

Comment:
The module is named Authentication because it covers everything related to authentication, not just a single action. Rails prefers naming modules after what they do overall, which makes the code easier to understand and maintain.

Rails conventionally prefers naming modules and concerns in a way that represents concepts or groups of behaviours. This makes the naming consistent with other modules such as Authorization, Logging, Monitoring, etc.

Besides, Writebook name their model Authentication.

![CleanShot 2024-07-20 at 17 57 13@2x](https://github.com/user-attachments/assets/907fc0cd-7876-4ba7-9a61-d8d08cb83932)